packages/vat-data/src/kind-utils.js

```diff
@@ -91,3 +91,8 @@ export const vivifySingleton = (
 };
 // @ts-expect-error TODO statically recognize harden
 harden(vivifySingleton);
+
+export const vivifyKind = (baggage, name, init, behavior) =>
+  defineDurableKind(provideKindHandle(baggage, name), init, behavior);
```

Comment:
I like it! Also, a more understandable delta from `defineKind`.

But you should still pass through the optional `options`

```suggestion
export const vivifyKind = (baggage, name, init, behavior, options = undefined) =>
  defineDurableKind(provideKindHandle(baggage, name), init, behavior, options);
```

Reply:
Done by #5772